src/web/components/Core/PagingToolHelper.spec.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  getPageNumberOptions,
+  getRowsPerPageOptions,
+  getShowingRowsText,
+} from './PagingToolHelper';
+
+describe('test paging tool helper functions', () => {
+  describe('check that correct rows per page options are returned', () => {
+    it('should return at least one rows per page option', () => {
+      expect(getRowsPerPageOptions(3)).toEqual([{ name: '10', id: 10 }]);
```

Comment:
I think this could benefit from some re-jigging. Could we extract out the expected result to a const and potentially parameterise the test and do some renaming - something like:
```
const singleRowsPerPageOption = [{ name: '10', id: 10 }];

test.each([[3], [1], [10]])(
  'should return a single rows per page option for %i total rows',
  (numberTotalRows) => {
    expect(getRowsPerPageOptions(numberTotalRows)).toEqual(singleOption);
  }
);
```